.github/workflows/release.yml

```diff
@@ -1,90 +1,70 @@
+name: Publish plerkle plugin 
+
 on:
+  workflow_dispatch:
   push:
     tags:
-      - "v*"
+      - 'v*'
 
 env:
   CARGO_TERM_COLOR: always
-  IMAGE_NAME: plerkle-test-validator
-  RUST_VERSION: 1.69.0
-  SOLANA_VERSION_STABLE: v1.16.6
+
 jobs:
-  release-stable:
-    runs-on: buildjet-8vcpu-ubuntu-2004
+  build20:
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+
+    runs-on: ["${{ matrix.os }}"]
+
     steps:
-      - uses: actions/checkout@v2
-      - name: Set env vars
-        run: |
-          source ci/env.sh
-          echo "GEYSER_PLUGIN_NAME=$plugin_name" | tee -a $GITHUB_ENV
-          echo "GEYSER_PLUGIN_LIB=lib${plugin_lib_name}" | tee -a $GITHUB_ENV
-      - if: runner.os == 'Linux'
-        run: |
-          sudo apt-get update
-          sudo apt-get upgrade
-          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-          sudo apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
-          sudo apt-get update
-          sudo apt-get install -y libudev-dev libssl-dev libsasl2-dev libzstd-dev
-          sudo apt-get install -y openssl --allow-unauthenticated
-          sudo apt-get install -y libssl1.1 --allow-unauthenticated
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ env.RUST_VERSION }}
-          override: true
-          profile: minimal
-          components: rustfmt
-      - name: Build Plugin
-        run: |
-          echo "CI_TAG=${GITHUB_REF#refs/*/}" >> "$GITHUB_ENV"
-          echo "CI_OS_NAME=linux" >> "$GITHUB_ENV"
-          cargo build --release
-      - name: Build release tarball
-        run: ./ci/create-tarball.sh
-      - name: Publish to crates registry
-        run: |
-          cargo publish -p plerkle_serialization --token $CARGO_TOKEN --no-verify || true
-          sleep 30
-          cargo publish -p plerkle_messenger --token $CARGO_TOKEN --no-verify || true
```

Comment:
We still need these crate publishes I believe, and I don't see them in the new code?

Reply:
Added back those lines